sirepo/package_data/static/en/landing.html

```diff
@@ -48,6 +48,7 @@
               <li><a href="/zgoubi">Zgoubi</a></li>
             </ul>
           </li>
+          <li><a href="/en/plans.html">Plans</a></li>
```

Comment:
For some reason, the underline stays after the click on contact and radiasoft home:

<img width="571" alt="Screen Shot 2020-07-02 at 8 41 17 AM" src="https://user-images.githubusercontent.com/5495179/86373675-e25e9680-bc40-11ea-86be-c5334f95c47d.png">

It is happening on sirepo.com so nothing you did, but it would be great if you could figure it out. Thanks.